src/routes/StackView/StackView.scss

```diff
@@ -1,17 +1,15 @@
 @import "constants/style";
 
-$stack-view__header-height: 16%;
-$stack-view__parent-note-height: 22%;
+$stack-view__header-height: 14vh;
```

Comment:
Is this supposed to be 14vh and not 16vh? (because header-height was 16% before)